backend/app/services/splunk.py

```diff
@@ -111,11 +111,19 @@ async def filterPost(self, query, searchList=""):
 
         try:
             # If additional search parameters are provided, include those in searchindex
-            search_query = ""
+            search_query = f"search index={self.indice} "
+
             if searchList:
-                search_query = f"search index={self.indice} {searchList} | stats count AS total_records, values(cpu) AS cpu, values(node_name) AS nodeName, values(test_type) AS benchmark, values(ocp_version) AS ocpVersion, values(ocp_build) AS releaseStream"
-            else:
-                search_query = f"search index={self.indice} | stats count AS total_records, values(cpu) AS cpu, values(node_name) AS nodeName, values(test_type) AS benchmark, values(ocp_version) AS ocpVersion, values(ocp_build) AS releaseStream"
+                search_query += f"{searchList} | "
+
+            search_query += (
```

Comment:
I like that you removed the redundancy here!

But ... the "not `searchList`" case also had a `|` after the index phrase (`search index={self.indice} |`), and the new version doesn't. Is that correct?

Reply:
So this now becomes `search index={self.indice} | {searchList} |`, with *two* pipes instead of one. I don't know the Splunk syntax -- is that OK?